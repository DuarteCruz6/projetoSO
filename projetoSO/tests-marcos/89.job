WRITE [(f7,68)(f0,40)(s8,41)(q7,8)]
WRITE [(b7,31)(w5,76)(h1,1)]
READ [h1]
WRITE [(g0,42)(m1,96)]
DELETE [h1,s8,f7,f0,m1]
WRITE [(c5,10)(m4,56)(b6,17)(m1,85)(z1,26)]
WRITE [(s2,0)(u2,73)]
WRITE [(v0,1)(g4,51)]
WRITE [(f8,49)]
WRITE [(y0,19)(e6,24)(a1,32)(j4,3)(o9,91)]
WRITE [(e5,11)(z9,87)(t3,83)(r5,4)(b5,6)]
READ [a1]
READ [g0]
WAIT 3000
WRITE [(a3,28)(b8,37)]
WRITE [(l4,71)(m0,32)(o6,69)(v8,87)]
WRITE [(c1,9)(j9,71)(a4,79)(u6,92)]
WRITE [(o5,20)(y9,49)(w8,94)(x6,23)]
WRITE [(i1,66)]
READ [u2]
DELETE [q7,u2]
WAIT 2000
WRITE [(n9,34)]
WRITE [(a8,33)(w0,40)(c8,5)]
WRITE [(t1,16)(k8,9)(s6,85)]
WRITE [(b4,18)(x8,100)]
READ [w0]
DELETE [e5,o6,a4,v0]
WRITE [(p2,32)(h0,22)(e8,15)]
WRITE [(a7,21)(h3,7)(j1,48)]
READ [s2]
WRITE [(c0,17)(j2,38)(b0,22)(x0,24)(g3,44)]
