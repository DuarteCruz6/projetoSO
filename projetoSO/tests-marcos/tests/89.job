WRITE [(i5,10)(e8,36)(d6,59)(t1,38)(b7,88)]
DELETE [t1,i5,d6,b7]
WRITE [(j4,51)(q0,63)]
WAIT 2000
BACKUP
WRITE [(p3,9)(b6,39)]
WRITE [(v1,7)(i6,33)(v1,50)(l6,78)]
SHOW
DELETE [q0,i6,b6,e8,p3]
DELETE [j4]
READ [l6]
WRITE [(f4,51)(n1,87)(v7,1)(n1,56)(w7,9)]
WRITE [(w0,62)]
