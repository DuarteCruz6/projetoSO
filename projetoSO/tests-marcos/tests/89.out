(b6, 39)
(e8, 36)
(i6, 33)
(j4, 51)
(l6, 78)
(p3, 9)
(q0, 63)
(v1, 50)
[(l6,78)]
